Applications/PaintBrush/main.cpp

```diff
@@ -53,7 +53,19 @@ int main(int argc, char** argv)
         }
         paintable_widget->set_bitmap(*bitmap);
     }));
+
     app_menu->add_separator();
+
+    NonnullRefPtr<GAction> eraser_color_toggler = GAction::create("Use secondary color for eraser", [&](const GAction&) {
+        bool toggled = !paintable_widget->eraser_use_secondary_color;
+        paintable_widget->eraser_use_secondary_color = toggled;
+        eraser_color_toggler->set_checked(toggled);
+    });
+
+    eraser_color_toggler->set_checkable(true);
+    eraser_color_toggler->set_checked(paintable_widget->eraser_use_secondary_color);
+    app_menu->add_action(*eraser_color_toggler);
```

Comment:
I think it would make more sense to add this as a context menu for the eraser tool button, like how the pen tool has a popup for the thickness.
(Actually, I'd like to expose those settings in a way that they're always visible on screen instead of hiding them in a menu, but at least we can be consistent :)